docs/api.rst

```diff
@@ -0,0 +1,97 @@
+API
+===
+
+This page describes module API with detailed explanations of the functions
+parameters.
+
+.. module:: aiohttp_jinja2
+.. highlight:: python
+
+
+.. function:: setup(app, *args, app_key=APP_KEY, context_processors=(),
+                    filters=None, default_helpers=True, **kwargs)
+
+   Function responsible for initializing templating system on application. This
+   one is required to be called first in order to start using this package with
+   your application.
+
+   :param app: :class:`aiohttp.web.Application` instance to initialize template
+               system on.
+
+   :param str app_key: key that will be used to access templating environment
+                       from application dictionary object.
+
+   :param context_processors: list of :ref:`aiohttp-web-middlewares`. These are
+                              context processors used to rewrite or inject some
+                              variables during the processing of a request.
+   :type context_processors: :class:`list`
+
+   :param filters: desc.
```

Comment:
Did you miss description?

Reply:
yes, it's still work in progress. I'm wondering whether it's something that was requested by the issue or you expected something else within the scope of this task?

Reply:
`__all__ = ('setup', 'get_env', 'render_template', 'template')`.
`get_env` and `template` should be documented (as well as installed context functions),
`render_string` should be added to `__all__`.